client/state/themes/constants.js

```diff
@@ -0,0 +1,10 @@
+/** @ssr-ready **/
+
+export const DEFAULT_THEME_QUERY = {
+	search: '',
+	tier: '',
```

Comment:
> '' - empty string is equivalent to `ALL`?

Yes. See e.g. [here](https://github.com/Automattic/wp-calypso/blob/master/client/my-sites/themes/theme-showcase.jsx#L59) (though it's admittedly not very clear from our code).